### PR TITLE
feat: Add REST API endpoints for setting up Access forms

### PR DESCRIPTION
**Pull request by RadovanTomik**
### Description:

Adds REST API endpoints for setting up Access forms. Test it using swagger.

### Checklist:

_Make sure you tick all the boxes bellow if they are true or do not apply:_

- [x] I have performed a self review of my code
- [x] My code follows [Google Java Code style](https://google.github.io/styleguide/javaguide.html)
- [x] I have made my code as simple as possible
- [x] I have added unit tests and the code coverage has not decreased
- [x] I have updated the documentation in all relevant places

### CODE REVIEW

File: src/main/java/eu/bbmri_eric/negotiator/dto/access_form/FormCreateDTO.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@JsonInclude(value = JsonInclude.Include.NON_NULL)
+public class FormCreateDTO {
```
Comment (svituz) [inline src/main/java/eu/bbmri_eric/negotiator/dto/access_form/FormCreateDTO.java:16]: AccessFormCreateDTO?

File: src/test/java/eu/bbmri_eric/negotiator/integration/api/v3/AccessFormTests.java [hunk 2]
```diff
@@ -35,6 +38,7 @@
 public class AccessFormTests {
 
   private static final String ENDPOINT = "/v3/access-criteria";
```
Comment (svituz) [inline src/test/java/eu/bbmri_eric/negotiator/integration/api/v3/AccessFormTests.java:40]: This should be unused now

**Review by svituz — COMMENTED** [review]
Apart from the minimal comments, it may be useful to have a functionality to insert elements and sections in a specific position, otherwise, if you want to change the form by adding one element or one section, you will need to restart from the beginning.

**Comment by RadovanTomik** [comment]
yes, you can do it using `src/main/java/eu/bbmri_eric/negotiator/dto/access_form/SectionLinkDTO.java` and `src/main/java/eu/bbmri_eric/negotiator/dto/access_form/ElementLinkDTO.java`. They have an attribute element and section order which sets the position

**Comment by svituz** [comment]
Is it implemented in the controller?

**Comment by RadovanTomik** [comment]
yes, the DTOs are passed for the link operation.

File: src/test/java/eu/bbmri_eric/negotiator/integration/api/v3/AccessFormTests.java (hunk 2 above)
Comment (RadovanTomik) [reply src/test/java/eu/bbmri_eric/negotiator/integration/api/v3/AccessFormTests.java]: removed

**Pull request merged**